src/components/molecules/VenueRoomItem/VenueRoomItem.tsx

```diff
@@ -60,10 +60,13 @@ export const VenueRoomItem: React.FC<VenueRoomItemProps> = ({
 
     const roomValues = getValues();
 
-    const roomUrl = window.origin + venueInsideUrl(roomValues.venueName);
+    const roomUrl =
+      window.origin +
+      venueInsideUrl(roomValues.venueName.replace(/\W/g, "").toLowerCase());
```

Comment:
It's worth to extract `roomValues.venueName.replace(/\W/g, "").toLowerCase()` as a separated var

Reply:
Or at least

```ts
const ALL_NON_WORDS = /\W/g
```

Reply:
This looks more like an already existing utility function that converts the input venue name to some canonical URL friendly form - a slug

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2382/commits/96aad0110710e24901a4de2ec5af7244d3b9d114